main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+
+    for player, info in players.items():
+        race = info["race"]
+        guild = info["guild"]
+
+        new_race = Race.objects.get_or_create(
```

Comment:
Using indexes is not very `pythonic`. Since this method returns a tuple you can use unpacking like so: `new_race, _`
This will make your code more respectable by other developers :wink: